docs/cli/cli-spans.md

```diff
@@ -0,0 +1,38 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: CLI
+--->
+
+# Semantic Conventions for CLI (Command Line Interface) programs
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions to apply when instrumenting CLI programs.
+operations to attributes on a Span.
+
+The **span name** SHOULD be of the format `{process.executable.name}`. In case this is
+not informative enough because of interpreters calling other files, e.g. `python script.py` or in
+cases of subcommands, e.g. `flask run mycommand` instrumentation MAY provide a more
+descriptive span name of the format `process.command_line` if appropriate.
+
+> **Warning**
+> The `process.command_line` value can have high cardinality.
```

Comment:
```suggestion
> The `process.command_line` value can have high cardinality in case of parameters passed by the user. It is recommended to strip parameters if there is enough domain knowledge for being able to do so.
```

Reply:
Good call, I think this is important. Could this be added to table below and as an update to the original `process` [registry](https://github.com/open-telemetry/semantic-conventions/blob/25e0baecd6393d10c1b87a7074520985f0da2fac/model/process/registry.yaml#L87-L95)?

Reply:
should this doc link to the process registry or is it assumed? personally, I didn't know there was one until you mentioned it..

Reply:
actually it is mentioned below, possibly handy to say up front that "a CLI span is defined in terms of process attributes" with a link to that?